Issues #5 and #6

drumrock opened the pull request:
- Added user.access method
- Added access.name method

drumrock commented:
4d04f60 wasn't supposed to be in this pull request.

mesilov commented on src/classes/user.php:
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace Bitrix24\User;
```
Can you move a class user into a folder? Without change a namespase becouse in other progects users can have an error if they load by full namespace.

drumrock commented:
I moved User class to subfolder and changed it's namespace back to Bitrix24\User.

Maybe we can add some 'duplicate' entity in correct location later.
Like Bitrix24\User\User class with empty body extending Bitrix24\User class.
For legacy code support, but with 'deprecated' comment.
